lib/assessment.js

```diff
@@ -5,7 +5,14 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  var result = arguments[0];
+  for (var i = 1; i < arguments.length; i++) {
+    if (arguments[i] < result) {
+      result = arguments[i];
+    }
+  }
 
+  return result;
```

Comment:
Spot on. This approach can be used to find the __est element in any set, so long as you can compare elements on a single value.